packages/core/handle/handle.js

```diff
@@ -6,6 +6,7 @@
  * function in the chain. If it returns `false` (or any falsy value like `null` or `undefined`),
  * the event handling chain stops at that input function.
  *
+ * @example
```

Comment:
These should not use `@example` or else they could be rendered into an executable code block in the docs.